src/main/kotlin/com/terraformation/backend/accelerator/model/ApplicationModel.kt

```diff
@@ -25,6 +30,38 @@ data class ExistingApplicationModel(
     val projectName: String,
     val status: ApplicationStatus,
 ) {
+  fun toGeoFeature(): SimpleFeature {
+    if (boundary == null) {
+      throw IllegalStateException("Application has no boundary")
+    }
+
+    val featureTypeBuilder = SimpleFeatureTypeBuilder()
```

Comment:
We did not use our types, and instead Primitive types here because I don't know how to pass our own JSON serializer to  `FeatureJSON`